executor/runtime/docker/docker.go

```diff
@@ -978,35 +978,39 @@ func (r *DockerRuntime) Prepare(ctx context.Context, pod *v1.Pod) error { // nol
 	logger.G(ctx).WithField("prepareTimeout", r.dockerCfg.prepareTimeout).Info("Preparing container")
 
 	var (
-		containerCreateBody container.ContainerCreateCreatedBody
-		myImageInfo         *types.ImageInspect
-		dockerCfg           *container.Config
-		hostCfg             *container.HostConfig
-		systemServices      []*runtimeTypes.ServiceOpts
-		size                int64
-		runTmpfs            string
-		err                 error
+		myImageInfo *types.ImageInspect
+		imageSize   int64
+		err         error
```

Comment:
It's better to use a named return value, so the code that returns from the function doesn't have to return this exact `err` variable for the defer to work.